framework/src/modules/interoperability/base_interoperability_endpoint.ts

```diff
@@ -154,6 +155,13 @@ export abstract class BaseInteroperabilityEndpoint extends BaseEndpoint {
 		};
 	}
 
+	// eslint-disable-next-line @typescript-eslint/require-await
+	public async getCCMSchema(_context: ModuleEndpointContext): Promise<Record<string, unknown>> {
```

Comment:
small correction, I think it will be ,
```suggestion
	public async getCCMSchema(_context: ModuleEndpointContext): Promise<{ schema: Record<string, unknown>}> {
```